_includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml

```diff
@@ -42,6 +42,10 @@ spec:
               value: "tigera-operator"
             - name: TIGERA_OPERATOR_INIT_IMAGE_VERSION
               value: {{.Values.tigeraOperator.version}}
+          envFrom:
```

Comment:
I didn't even know `envFrom` was a thing. "The more you know" :rainbow:

In general I think it would work fine this way, my only reservation might be that if we used 2 `env` and made them optional then we would have the env var names defined there and it might be a little more self documenting. WDYT?

Reply:
+1 for "more self documenting" although @fasaxc :
> envFrom has the advantage that it's 100% certain that, if the config map is not there` then we can't interfere with KUBERNETES_SERVICE_HOST/PORT (even if k8s changes its implementation later).  WDYT?

Reply:
Ok yeah I see that the API isn't clear on what is done with an Env Var that has an optional ConfigMap. I'm good with this to ensure we don't have env vars if we don't need them.